Scripts/static-analyzer/Sources/Analyzer/SourceNavigator+SymbolLookup.swift

```diff
@@ -0,0 +1,197 @@
+import Foundation
+import IndexStoreDB
+import TSCBasic
+
+extension SourceNavigator {
+
+    /// Similar to Xcode's "Go to definition" feature, this function can be used to find the code that defines a type
+    /// or a function that's represented by the given USR.
+    public func definition(ofUSR usr: USR) -> Symbol? {
```

Comment:
Nitpick, thanks to the dedicated `USR` type, this method name doesn't need to compensate for weak type information with a label parameter:


```diff
-    public func definition(ofUSR usr: USR) -> Symbol? {
+    public func definition(of usr: USR) -> Symbol? {
```